### ISSUE TITLE
Finalise service rules & justifications for the lead provider API

### ISSUE THREAD

**Issue by claire-hughez** [DFE-Digital/ecf2#12]
### Why

In the discovery, we identified that some of the validation rules over the Lead Provider API are not quite right. They might be overly restrictive, or there might not be validation where they needs to be.

We still need to understand better what validation rules exist over the Lead Provider API, so then we and the LPDOB can suggest improvements.

### What

Get from LPDOB team list of validation rules for Lead provider API

Get their understanding of what validation rules might need to be improved

### Context & links

https://lucid.app/lucidspark/3edf8820-f543-46b2-9041-410bdbee484b/edit?invitationId=inv_690d31d3-e4fc-4591-b5ee-2a5e5e0afdf5&page=0_0#

### Who knows about this

@claire-hughez @mason-emily

**Comment by kas-madrigal** [comment]
@claire-hughez can I archive this ticket? It's showing up in AB stuff I'm not sure why and I think it's old and done

**Comment by claire-hughez** [comment]
No! Sorry. We still need to do it when LPDOB frees up. It doesn't have a squad so shouldn't be in AB bits? @kas-madrigal
